lib/actions-oclif/env/add.ts

```diff
@@ -91,9 +90,7 @@ export default class EnvAddCmd extends Command {
 		},
 	];
 
-	// hardcoded 'env add' to avoid oclif's 'env:add' topic syntax
-	public static usage =
-		'env add ' + new CommandHelp({ args: EnvAddCmd.args }).defaultUsage();
+	public static usage = 'env add <name> [value]';
```

Comment:
These were changed for consistency with other commands.  We can later look at autogenerating these.